common/model/cpptransaction.cc

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
```

Comment:
Why didn't copyright owner belong to IOTA Foundation?

Reply:
The correct legal name is `IOTA Stiftung`. `IOTA Foundation` is just the english translation of that. Both work arguably.